.github/workflows/ci-format.yml

```diff
@@ -36,4 +36,35 @@ jobs:
             -s n \
             --ignore marl_benchmark,examples,scenarios,docs,manager_pb2_grpc.py,worker_pb2_grpc.py \
             --msg-template='{path}: line {line}: {msg_id}: {msg}' \
-            ./smarts ./envision
\ No newline at end of file
+            ./smarts ./envision
+
+  test-types:
+    runs-on: ubuntu-18.04
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+    container: huaweinoah/smarts:v0.4.18-minimal
+    steps:
+      - name: Install packages
+        run: |
+          add-apt-repository -y ppa:git-core/ppa
+          apt-get update
+          apt-get install -y git
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install SMARTS
+        run: |
+          python3.7 -m venv ${{env.venv_dir}}
+          . ${{env.venv_dir}}/bin/activate
+          pip install --upgrade pip wheel
+          pip install .[camera-obs,dev,train]
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v13.2  
+        with:
+          base_sha: origin/develop     
```

Comment:
This is run only on `develop` branch or on any branch which is based on develop during pull request?

Reply:
This pytype check runs on all branches, and takes the git diff with the develop branch for type checking.